fauna/README.md

```diff
@@ -0,0 +1,39 @@
+# Agent Check: Fauna
+
+## Overview
+
+This integration allows Fauna users to import their database query logs into Datadog. Users are able to configure it for specific databases or for entire region groups. When enabled all requests made to fauna matching the database or region group configuration will show up in Datadog logs under the fauna service.
+To see what fields are included, see the [Fauna Query Log Record Format][1].
```

Comment:
The overview should include: 
	1	What does your product do in 1-2 sentences?
	2	What will customers get from this integration (the value of the integration & why it's helpful)
	3	What specific data your integration monitors and the value?